app/controllers/case_workers/admin/allocations_controller.rb

```diff
@@ -6,6 +6,7 @@ class CaseWorkers::Admin::AllocationsController < CaseWorkers::Admin::Applicatio
   before_action :set_summary_values, only: [:new], if: :summary_from_previous_request?
   before_action :process_claim_ids, only: [:create], if: :quantity_allocation?
 
+  # FIXME: is this even used....the partial it is called from does not appear to be used!!
```

Comment:
Maybe not... check with @cleargif it may have been a legacy helper from the old allocation page... or still used on re-allocation